### ISSUE TITLE
Cannot download mods (Error 403)

### ISSUE THREAD

**Issue by yalexashkin** [cdbbnnyCode/modpack-installer#13]
Seems like something is wrong with curseforge api
Script spams with this message(but different project IDs):
```
Error 403 trying to access https://addons-ecs.forgesvc.net/api/v2/addon/441706

fetch failed
```

**Comment by cdbbnnyCode** [comment]
Looks like CurseForge updated their API. More research needed to figure out how to download mod info in a more up-to-date, legitimate manner.

**Comment by rdu** [comment]
Have you made any progress on this? My son is eager to install a specific mod. I don't want to install windows though.

**Comment by yalexashkin** [comment]
I've found this project https://git.sakamoto.pl/domi/curseme
They are stealing api key from official launcher and then using it in requests

**Comment by cdbbnnyCode** [comment]
I might be able to get a legitimate CF Core API key rather than scraping it from the official installer. More research is needed, however.

**Comment by cdbbnnyCode** [comment]
CurseForge finally sent an API key; next step is to rewrite the entire request system to use it.

**Comment by cdbbnnyCode** [comment]
Fixed in [c6a61a8](https://github.com/cdbbnnyCode/modpack-installer/commit/c6a61a85cf32bae79850168b55eef52f07334fb2). The project now uses the official CF API.
